README.md

```diff
@@ -283,27 +283,31 @@ result = entrez_api.search(
     database='pubmed',
     max_results=1
 )
-result.data['esearchresult']['idlist']
+try:
+    print(result.data['esearchresult']['idlist'])
+except KeyError as exc:
+    raise ValueError(
+        f"Unexpected response data blob {result.data}."
+    ) from exc
```

Comment:
I would skip error handling from the examples in the readme. They are meant to be illustrative, nor comprehensive.

Reply:
Good point, thanks!  Will implement that shortly